### ISSUE TITLE
autoplay button sticking out of the player style

### ISSUE THREAD

**Issue by zaxel-github** [lightbeam24/CustomTube#77]
![Screenshot 2023-10-06 002232](https://github.com/lightbeam24/CustomTube/assets/79190563/3ddfd6dd-adbe-4771-9e14-2e056afbb7f3)
![Screenshot 2023-10-06 002123](https://github.com/lightbeam24/CustomTube/assets/79190563/2ad9969a-77e7-4aa1-93ee-7c9f8f9c6f60)
![Screenshot 2023-10-06 002306](https://github.com/lightbeam24/CustomTube/assets/79190563/9b01311e-5466-49bc-9c66-8054072e90f4)
this effects all the player styles that arent the newest one. it probably happens because of the smaller formfactor these player bars took up, or that the autoplay button wasnt there back then. i really dont know.

**Comment by lightbeam24** [comment]
It's because it wasn't there back then. Will look into it eventually, but I can't say when.

**Comment by ArbestRiAgain** [comment]
YouTube Redux has the autoplay button where it's supposed to be

**Comment by lightbeam24** [comment]
Yes I know. I eventually plan to do that as well, but it hasn't been a priority yet. Now that you've mentioned it though, I'll try to get to it sooner.

**Comment by bangbangyouall** [comment]
use ublock to disable the autoplay button like i did

**Comment by uy813** [comment]
> use ublock to disable the autoplay button like i did
ublock is not the extension for old youtube layouts. it is the extension to block ads
